runtime/src/main/java/org/corfudb/runtime/ClusterBootstrapUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.corfudb.runtime;
+
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.clients.IClientRouter;
+import org.corfudb.runtime.clients.LayoutClient;
+import org.corfudb.runtime.clients.ManagementClient;
+import org.corfudb.runtime.clients.NettyClientRouter;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.NodeLocator;
+import org.corfudb.util.Sleep;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Utility to bootstrap a cluster.
+ *
+ * <p>Created by zlokhandwala on 1/19/18.
+ */
+@Slf4j
+public class ClusterBootstrapUtil {
+
+    @Getter
+    @Setter
+    private static int retries = 3;
+
+    @Getter
```

Comment:
this seems a bit weird, maybe just pass it through the method signature

Reply:
Done.